### ISSUE TITLE
[docs] Revise the Getting Started docs

### ISSUE THREAD

**Issue by markcaron** [RedHat-UX/red-hat-design-system#1176]
### Discussed in https://github.com/orgs/RedHat-UX/discussions/1150

<div type='discussions-op-text'>

<sup>Originally posted by **nikkimk** July 19, 2023</sup>
@Nouveau and I were discussing the **Getting Started** docs and wondered if we need to think about the audience for the docs, as we have several audiences to consider: design system consumers versus contributors and designers versus developers.

We took a look at how other design system docs are organized:
- [Carbon](https://carbondesignsystem.com/) organizes into **Designing**, **Developing**, **Contributing**, and **Migrating**
- [Patternfly](https://www.patternfly.org/) organizes into **Contribute->Design**, **Contribute->Develop**, **Get Started->Contribute**, and **Get Started->Design and Develop** under it.
- [Patternfly elements](https://patternflyelements.org/) organizes into **Get started** and **Develop**
- [Material design]() organizes into **Get started->Design**, **Get started->Develop**, and **Develop**

We're thinking we should reorganized this content something like this:
- Get started
   - For designers
     - Overview
     - Design system kit
     - FTS
     - Icons, etc.
  - For developers
     - Overview
     - Using npm
     - Using CDN**
     - React wrappers* coming soon
     - Vue wrappers* coming soon
- Contribute***
   - Overview
   - Bugs and requests
   - Documentation
   - Components
   - Icons
   - Patterns
  
**Link to VPN for Red Hat-specific Implementation
***Contribute page* see Carbon’s contributing section

</div>

**Comment by bennypowers** [comment]
Follow up from Aug 7 office hours:
1. Develop a graphical motif to represent the concepts 'token', 'element', and 'pattern'
  - these graphical identities should be legible at large (landing page) and small (header nav) sizes 
  - think "atomic design" with h/t to Brad Frost
1. Add a "Components" page to the top-level nav
  - will be a landing page with three graphic cards briefly explaining and linking to tokens, elements, patterns
1. Add graphic cards with links and motifs to "get started" with explanations of 'tokens',  'elements', and 'patterns'
1. Add breadcrumb-like "you are here" maps to each page's header
  - utilizing the graphical motif developed in step 1

In addition, we should add a page (perhaps under foundations? but the exact location TBD) with detailed guidance on colour context

cc @coreyvickery for your consideration, and with thanks to @nikkimk and @markcaron for leading the conversation

**Comment by bennypowers** [comment]
one idea for a strech goal:

In addition to the three cards on `/components`, we could also offer a decision tree ui, like

"I want to [build a page|contribute to rhds]" > "I need a [small ui component|complex ui item]" > "I need a [form control|layout container]" > rh-card docs

**Comment by markcaron** [comment]
Breaking down Tokens, Elements, Patterns as our "ATOMIC" building blocks:

![image](https://github.com/RedHat-UX/red-hat-design-system/assets/1890950/a1569d2f-375b-4b0d-9149-1217befcc08c)

**Comment by marionnegp** [comment]
> Add detailed explanations of the concepts 'token', 'element', and 'pattern' to their respective overview pages

Would this info work better under the About section? I see tooling info going in the Get Started section and any conceptual info under About. Of course, we could tell people to check the About pages for more info about our terminology.

**Comment by marionnegp** [comment]
@coreyvickery and I are updating the [Color page](https://ux.redhat.com/foundations/color/). Do you think it makes sense to add there with theming information?

**Comment by bennypowers** [comment]
in general i think the balance between designer-oriented and engineer/front-end-oriented content on ux is off.

We need to set the balance so that both groups can easily find the info they need.

Use of color as a design component while it is a front-end concern, IMO it makes more sense for designers to make those decisions higher up the chain. our goal should be to reduce the number of occasions in which front-end developers have to think about specific colour *values* as much as possible. Instead, front-end devs should be thinking in terms of *named semantic colour tokens* 

From that perspective, we need *comprehensive docs* on our colour context system

**Comment by marionnegp** [comment]
I've started writing content for the Designers page in [this Google doc](https://docs.google.com/document/d/1UjPUAj7bALF3jAng5NeSPmvgRUfYmo0t_jftDYQQTWk/edit?usp=sharing).

**Comment by marionnegp** [comment]
@RedHat-UX/developers, I started a [Google doc](https://docs.google.com/document/d/15AoxTvIfWi6RyUZAKaba-wXzISvzVS8XmsJ8W7XiFj0/edit?usp=sharing) for the Get Started: Developers page. Most of this info is copied from our README files and lightly edited. I've added highlighted placeholder text to call out sections that need content.

**Comment by markcaron** [comment]
@nikkimk @bennypowers @brianferry do you all mind looking at Marionne's doc above and providing feedback?

**Comment by marionnegp** [comment]
The [Overview](https://ux.redhat.com/get-started/) and [Designers](https://ux.redhat.com/get-started/designers/) pages are live. The Developers page is in a [draft PR](https://github.com/RedHat-UX/red-hat-design-system/pull/1439).

**Comment by marionnegp** [comment]
@markcaron @hellogreg, Get Started pages for designers and developers are up. Should I keep this issue open until we have a page for contributors?

**Comment by marionnegp** [comment]
@markcaron, since we're just waiting on contributor docs, can I close this in favor of #1175?

**Comment by bennypowers** [comment]
Closing in favour of #1175
